pkg/olm/clusterserviceversion.go

```diff
@@ -3,6 +3,7 @@ package olm
 import (
 	"context"
 	"fmt"
+
```

Comment:
Accident?

Reply:
no, there is no explanation, but lint verification fails on fmt for clusterserviceversion, so I have to fix it each time